### PR TITLE
contributing: add PR filing process

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? 

Add PR filing process to `CONTRIBUTING.md`

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -120,3 +120,82 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
...
+> 目前 TiDB 主要维护三个版本的文档：dev（最新开发版），v3.0（最新稳定版），v2.1（最新 2.1 版）。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。
+
+### 第 1 步：Fork pingcap/docs-cn 仓库
+
+1. 打开 pingcap/docs-cn 项目[仓库](https://help.github.com/articles/github-glossary/#repository)：<https://github.com/pingcap/docs-cn>
+2. 点击右上角的 [**Fork**](https://help.github.com/articles/github-glossary/#fork) 按钮，等待 Fork 完成即可。
+
+### 第 2 步：将 Fork 的仓库克隆至本地
+
+```
```
Comment (yikeke) [inline CONTRIBUTING.md]:
```suggestion
```shell
```
要不要加个bash或shell高亮一下代码？

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -120,3 +120,82 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
 26. [MD046 - Code block style](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md046---code-block-style)
 
     文档中代码块统一使用**三个反引号**进行包裹，**禁止**使用**缩进四格**风格的代码块。(`style`=`fenced`)
+
+## Pull Request 提交流程
```
Comment (yikeke) [inline CONTRIBUTING.md]:
```suggestion
## Pull Request 提交流程
```
我觉得这一块放到【必须遵循的 markdownlint 规则】前面去比较好，因为要先提PR，才有可能碰到markdownlint check

Comment (lilin90) [reply CONTRIBUTING.md]: @yikeke But it would be better if they know the rules before they file PRs.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lilin90) [reply CONTRIBUTING.md]: @yikeke This doc won't be displayed on the website.

File: CONTRIBUTING.md (hunk 2 above)
Comment (yikeke) [reply CONTRIBUTING.md]: Ok.

**Pull request merged**